stockflux-launcher/public/window.css

```diff
@@ -0,0 +1,127 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,100,300,500);
+
+body {
+  margin: 0;
+  border: 1px solid #1A212A;
```

Comment:
We've got variables for the colours in this file

Reply:
Unless they don't work in the public folder

Reply:
They don't work in the public folder.